### PR TITLE
Random Names from Team

### PR DESCRIPTION
**Pull request by nevespt**
[EDITED]
- Added a function to read from Credits.txt and add a random name for the "Empty" Player Name
- Change the "BOT" text to blue color

[CHANGED]
- Changed Location of Credits.txt from '/Assets' to '/Assets/Resources'

### CODE REVIEW

File: Assets/Script/PlayerManager.cs [hunk 1]
```diff
@@ -4,21 +4,72 @@
...
+
 		public struct LastScore {
 			public DiffPercent percentage;
 			public DiffScore score;
 			public int notesHit;
 			public int notesMissed;
+
+		}
+
+		public struct randomName {
```
Comment (EliteAsian123) [inline Assets/Script/PlayerManager.cs:21]: Please follow C# naming conventions (should be `RandomName`)

**Review by EliteAsian123 — CHANGES_REQUESTED** [review]
Thanks for your PR, it's appreciated as always! Please look at review comments, and then I can merge.

File: Assets/Script/PlayerManager.cs (hunk 1 above)
Comment (nevespt) [reply Assets/Script/PlayerManager.cs]: all good now, thanks for the naming guideline <3

**Comment by nevespt** [comment]
ig its all good now, sorry about the naming

**Review by EliteAsian123 — APPROVED** [review]

**Pull request merged**